applications/minotari_ledger_wallet/common/Cargo.toml

```
@@ -9,3 +9,4 @@ license.workspace = true
 
 [dependencies]
 bs58 = { version = "0.5.1", default-features = false, features = ["alloc"] }
+serde = { workspace = true, features = ["derive"] }
```

With serde = "1" in the workspace, the dependency will use the default features (`default = ["std"]`).
```suggestion
serde = { workspace = true, default-features = false, features = ["derive"] }
```